### PR TITLE
feature(ui-ux): dex pair type filter and search box implementation

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### What kind of PR is this?:

/kind feature

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/components/HeaderSearchInputV2.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      >
+        <ThemedIcon
+          light={tailwind('text-mono-light-v2-900')}
+          dark={tailwind('text-mono-dark-v2-900')}
+          iconType='Feather'
+          name='chevron-left'
+          size={24}
+        />
+      </TouchableOpacity>
+      <SearchInputV2
```
Comment (kyleleow) [inline mobile-app/app/components/HeaderSearchInputV2.tsx:44]: can you check the font color in the text field? currently it is not `light-v2-900`
IMG_2DEB32673828-1

File: mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx [hunk 2]
```diff
@@ -193,24 +250,50 @@ export function DexScreen (): JSX.Element {
...
-            testID='dex_tabs'
-            lightThemeStyle={tailwind('bg-transparent')}
-            darkThemeStyle={tailwind('bg-transparent')}
-          />
-        </View>
-      </ThemedViewV2>
-      <TopLiquiditySection onPress={onSwap} pairs={topLiquidityPairs} />
-      <NewPoolsSection onPress={onAdd} pairs={newPoolsPairs} />
+    <ThemedViewV2
+      light={tailwind('bg-mono-light-v2-100')}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx:254]: can skip this, default color

Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Dex/DexScreen.tsx:254]: default color
```suggestion

```

File: mobile-app/app/components/HeaderSearchInputV2.tsx (hunk 1 above)
Comment (fullstackninja864) [reply mobile-app/app/components/HeaderSearchInputV2.tsx:44]: updated

**Pull request merged**